modules/core/arrow-core-data/src/main/kotlin/arrow/typeclasses/EqK.kt

```diff
@@ -26,4 +26,28 @@ interface EqK<F> {
    * ```
    */
   fun <A> Kind<F, A>.eqK(other: Kind<F, A>, EQ: Eq<A>): Boolean
+
+  /**
+   * Lifts the equality check provided by an given Eq instance
```

Comment:
Lift into what? Something like lifts to `Kind<F, A>` or similar might make sense